Dockerfile

```diff
@@ -65,18 +65,17 @@ RUN set -e \
     go get -u -v github.com/josharian/impl && \
     go get -u -v github.com/haya14busa/goplay/cmd/goplay && \
     go get -u -v github.com/davidrjenni/reftools/cmd/fillstruct && \
-    go get -u -v github.com/alecthomas/gometalinter && \
     go get -u -v github.com/go-delve/delve/cmd/dlv && \
     go get -u -v github.com/rogpeppe/godef && \
     go get -u -v github.com/uudashr/gopkgs/cmd/gopkgs && \
     go get -u -v golang.org/x/tools/cmd/gotype && \
     GO111MODULE=on go get -v golang.org/x/tools/gopls@latest && \
     go build -o /go/bin/gocode-gomod github.com/stamblerre/gocode && \
-    gometalinter --install && \
     chmod -R 777 "$GOPATH" && \
     apk del .build-deps && \
     mkdir /.cache && chmod -R 777 /.cache && \
-    apk add gcc g++ git
+    apk add gcc g++ git && \
```

Comment:
Adding these dependencies should be done with the rest of the `apk add` (lines 24-30); `git` is already in that list.

Reply:
these were already in the existing 1.12.9 Dockerfile. Unclear why they're being RE-added this late in the process when they were all added above already (except g++). Anyway, replacement PR is https://github.com/che-dockerfiles/che-sidecar-go/pull/4/files